miqi/documents/document_parser.py

```
@@ -357,74 +357,52 @@ def _parse_pdf(
-                        ["tesseract", str(img_path), "stdout", "-l", "chi_sim+eng"],
-                        capture_output=True, text=True, timeout=30,
-                        env=_env,
-                    )
-                    if result.returncode == 0 and result.stdout.strip():
-                        full_text_parts.append(result.stdout.strip())
+                    result, _ = engine(img)
+                    if result:
+                        lines = [line[1] for line in result if len(line) > 1 and line[1].strip()]
+                        if lines:
+                            full_text.append("\n".join(lines))
                 except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)